tests/dataset/test_aoi.py

```diff
@@ -81,6 +81,65 @@ def test_bounds_iou(self) -> None:
         expected_iou = 0.013904645827033404
         assert iou == expected_iou
 
+    def test_corrupt_raster(self) -> None:
+        raster_file = "tests/data/massachusetts_buildings_kaggle/22978945_15_uint8_clipped_clipped.tif"
```

Comment:
Where does this imagery file come from? It's not added in PR and shouldn't already be in the zip files with data. Also, it would be important that the filename contains "corrupted" somewhere.